sale_outplacement/models/outplacement.py

```diff
@@ -130,16 +130,21 @@ def suborder_process_data(self, data):
         partner_id = self._get_partner_id(data)
 
         skill = self._get_skill(data)
+        order_lines = [
+            (0, 0, {"product_id": self.env.ref("sale_outplacement.startersattning").id}),
+            (0, 0, {"product_id": self.env.ref("sale_outplacement.slutersattning").id}),
+        ]
         order = self.env['sale.order'].create({
             'origin': data['genomforande_referens'],
             'name': data['ordernummer'],
             'partner_id': partner_id,
+            'order_line': order_lines,
         })
-        product = self.env.ref('sale_outplacement.product_suborder')
-        self.env['sale.order.line'].create({
-            'product_id': product.id,
-            'order_id': order.id,
-        })
+        # product = self.env.ref('sale_outplacement.product_suborder')
```

Comment:
Is this to be kept, if so add comment on why

Reply:
solved